### ISSUE TITLE
Forms - Required fields

### ISSUE THREAD

**Issue by anormand** [marcom-unimelb/unimelb-design-system#185]
Required fields should be indicated using aria-required rather than data-required
http://www.w3.org/TR/WCAG20-TECHS/ARIA2.html

**Comment by waitingallday** [comment]
Fixed in https://github.com/marcom-unimelb/unimelb-design-system/commit/83960e786ac07bc23c3ff0e13cba3e006da65cea
